### ISSUE TITLE
Guests turnover not working correctly?

### ISSUE THREAD

**Issue by ryanbowden** [OpenRCT2/OpenRCT2#2010]
I Have build a park and set it hard to attract new guests (I like a challenge)

But I have noticed peeps in my park that have been there for 12 Hours (I play in fast mode a lot)

This means despite having a £50 entrance fee, and plenty of rides and around 1,500 guests the park struggles to make any profit.

I have an idea that the guests are not leaving correctly meaning the game not adding any new guest my game save is here
http://1drv.ms/1Nbfz2J

you should be able to download it. 

Will be interesting if this is a bug.

**Comment by Gymnasiast** [comment]
This sounds like intended behaviour to me. You have reported intended behaviour before. Also, you haven't checked (or not reported) the behaviour in RCT2.

**Comment by ryanbowden** [comment]
Hmm,

I think I now what the bug is, I just ran that save and removed all the bench's, the peeps energy stays maxed out I ran it for two year still all peeps energy still max.

I KNOW this is not intended behaviour as when playing rct2 the game will have tried peeps walking around all the time mine doesn't everyone is fuklly max it with full enger. this must be why peeps are staying in the park to long.

I can't test the origin RCT2 as it crashes when I click to select a saved game. Meaning I can't test it.
